src/sctools/groups.py

```diff
@@ -4,19 +4,21 @@
 """
 
 from crimson import picard
-import argparse
 import os
 import pandas as pd
 
 
-def aggregate_picard_metrics_by_row(file_names, output_name):
-    """
-    pipeline outputs picard QC metrics at single cell/sample level.
-    This function is called to merge/aggregate QC metrics by metrics type and
-    then merge multiple QC measurement into single matrix file.
-    In this file, column is sample/cell and row is QC metrics
-    :param file_names: list of files generated by Picard
-    :param output_name: output name without suffix'
+def write_aggregated_picard_metrics_by_row(file_names, output_name):
```

Comment:
These are close, but you can pattern-match on the existing docs: https://github.com/HumanCellAtlas/sctools/blob/master/src/sctools/barcode.py#L37